package laboratorio4.exercicios;


/*
Crie um recurso personalizado chamado RecursoExercicio2 que em seu método close imprima a seguinte mensagem: "Meu primeiro recurso personalizado do try-with-resouce",
este recurso deve possuir também um método chamado executaAlgo(), este método deve imprimir a mensagem: "Executando alguma ação que pode retornar uma exception!", este método
deve lançar um Exception na assinatura do método com a declaração throws.
* */

public class RecursoExercicio2 {

    // Realize a implementação aqui!

}
